README.md

```diff
@@ -248,7 +251,87 @@ DELETE `/v1/datasync/{account}/movers/{group}/{name}`
 | **404 Not Found**             | account not found               |
 | **500 Internal Server Error** | a server error occurred         |
 
+### Run a Data Mover Task
 
+POST `/v1/datasync/{account}/movers/{group}/{name}/{start}`
+
+| Response Code                 | Definition                      |
+| ----------------------------- | --------------------------------|
+| **200 OK**                    | start the data mover            |
+| **400 Bad Request**           | badly formed request            |
+| **404 Not Found**             | account not found               |
+| **500 Internal Server Error** | a server error occurred         |
+
+#### Example list response
+
+```json
+[
+    "TaskExecutionArn": "arn:aws:datasync:us-east-1:516855177326:task/task-05cd6f77d7b5d15ac/execution/exec-086d6c629a6bf3581"
```

Comment:
This should just return the id of the run that was created, e.g. `"exec-086d6c629a6bf3581"`

Reply:
Added code to show only id --- e.g.: "exec-086d6c629a6bf3581"